src/Core/Components/DateTime/FluentTimePicker.razor.cs

```diff
@@ -26,7 +26,7 @@ protected override bool TryParseValueFromString(string? value, out DateTime? res
         }
         else
         {
-            result = Value?.Date;
+            result = string.IsNullOrWhiteSpace(value) ? null : Value?.Date;
```

Comment:
Could you use something like that, to avoid to call `DateTime.TryParse` for empty string.

```csharp
if (string.IsNullOrWhiteSpace(value)) 
{
    result = null;
}
else if (value != null && DateTime.TryParse(value, out var valueConverted))
{
    result = currentValue.Date + valueConverted.TimeOfDay;
}
else
{
    result = Value?.Date;
}
```